corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
we should almost always avoid subqueries, and especially avoid filtering or joining on a result from a subquery. to get around that, we can define any number of intermediate tables which we first load and then join to each other or do any other filtering from